server/service.go

```diff
@@ -277,22 +269,6 @@ func (m *BoostService) sendValidatorRegistrationsToRelayMonitors(payload []build
 	}
 }
 
-// func (m *BoostService) sendAuctionTranscriptToRelayMonitors(transcript *AuctionTranscript) {
```

Comment:
if you removed the AuctionTranscript type, it makes sense to also remove this

Reply:
This has been removed, what do you mean?

Reply:
Ah yeah my bad I thought the comment was kept, all good